packages/core/src/modules/slack/use-cases/update-birthdates-to-db.ts

```diff
@@ -0,0 +1,30 @@
+import { db } from '@/infrastructure/database';
+import { slack } from '@/modules/slack/instances';
+
+export async function updateBirthdatesfromSlack() {
+  const members = await db
+    .selectFrom('students')
+    .select(['slackId'])
+    .where('slackId', 'is not', null)
+    .execute();
+
+  for (const member of members) {
+    await updateBirthdatebySlackId(member.slackId!);
+  }
```

Comment:
My concern/question is if we need to use a rate limiter here. I noticed @omar-elbaz had used one in a previous PR but here we are only retrieving the user profiles one at a time. Since ultimately the migration will only occur once it might be simple enough to implement rate limiting in line with the info from the [Slack docs](https://api.slack.com/apis/rate-limits#overview).